include/variables.h

```diff
@@ -1390,12 +1390,12 @@ extern UNK_PTR D_801C0B0C;
 // extern UNK_TYPE1 D_801C0EA0;
 // extern UNK_TYPE1 D_801C0EAC;
 // extern UNK_TYPE1 D_801C0EB8;
-// extern UNK_TYPE2 D_801C0EC0;
-extern s16 sQuakeRequestCount;
-extern quake_callback_func sQuakeCallbacks[7];
-extern s16 D_801C0EE4;
-extern s16 D_801C0EE8;
-extern s16 D_801C0EEC;
+// extern s16 sIsQuakeInitialized;
+// extern s16 sQuakeRequestCount;
+// extern quake_callback_func sQuakeCallbacks[7];
+// extern s16 D_801C0EE4;
+// extern s16 D_801C0EE8;
+// extern s16 D_801C0EEC;
```

Comment:
Those variables seems to be decomped by this PR. If they aren't supposed to be visible to other files, then it would be preferable to remove them instead of simply commenting them.